BAZProject/MainMovie/MainMoviePresenter.swift

```diff
@@ -0,0 +1,25 @@
+//
+//  MainMoviePresenter.swift
+//  BAZProject
+//
+//  Created 1030364 on 18/10/22.
+//  Copyright © 2022 ___ORGANIZATIONNAME___. All rights reserved.
+//
+//  Template generated by Juanpe Catalán @JuanpeCMiOS
+//
+
+import UIKit
+
+class MainMoviePresenter: MainMoviePresenterProtocol {
```

Comment:
Considera marcar como final todas las clases para el próximo PR
```suggestion
final class MainMoviePresenter: MainMoviePresenterProtocol {
```